opentelemetry/Cargo.toml

```diff
@@ -21,17 +21,17 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-async-std = { version = "1.6", features = ["unstable"], default-features = false, optional = true  }
+async-std = { version = "1.6", features = ["unstable"], default-features = false, optional = true }
 async-trait = { version = "0.1", optional = true }
 bincode = { version = "1.2", optional = true }
 dashmap = { version = "4.0.0-rc6", optional = true }
 fnv = { version = "1.0", optional = true }
-futures = "0.3"
+futures = { version = "0.3", features = ["std"] }
```

Comment:
isn't `std` a default feature? did you want to have `default-features = false`? or was this for something else?

Reply:
Cannot really recall why I changed that. But seems unnecessary. Will revert this part.